### ISSUE TITLE
Targetting .net core

### ISSUE THREAD

**Issue by Slesa** [boo-lang/boo#219]
Hi folks,
as we have no .NET5  version, maybe we can get a .NET6 one? 
There are a few options:
- how does IronPython the job?
- how does roselyn work?
- trying out [MetadataBuilder](https://learn.microsoft.com/en-us/dotnet/api/system.reflection.metadata.ecma335.metadatabuilder?view=net-7.0)
or any combination of it...

Did someone try something like this already? It would be such a pity to loose boo (as I did not have a chance to try it out...)

**Comment by RealDoigt** [comment]
Yes, there is very very slow, but ongoing work on this. Check Mason Wheeler's and Boggs branches/forks of this repo. Currently there's a bug that prevents debugging which ultimately also prevents progress on this issue.

**Comment by mattmc3** [comment]
We just merged a .NET 10 build, so Boo's back, baby!

**Comment by masonwheeler** [comment]
Completed via #222.
